api/src/main/java/org/jboss/arquillian/graphene/GrapheneElementAction.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat, Inc. and individual contributors
```

Comment:
If we really need to have license header in each file let's make the date properly generated. But I think we only need LICENSE file in the root.